packages/inter-protocol/src/vaultFactory/vaultHolder.js

```diff
@@ -58,23 +52,19 @@ export const prepareVaultHolder = (baggage, marshaller) => {
       holder: HolderI,
     },
     /**
+     * Init state of new Vault Holder
      *
      * @param {Vault} vault
-     * @param {ERef<StorageNode>} storageNode
+     * @param {StorageNode} storageNode
      * @returns {State}
      */
     (vault, storageNode) => {
       /** @type {PublishKit<VaultNotification>} */
       const { subscriber, publisher } = makeVaultHolderPublishKit();
```

Comment:
I haven't looked, but please ensure that `publisher.finish()` or `publisher.fail()` is called when the vaultHolder should store its final value and be unpinned for GC.  This should be reviewed for all callers of `pipeTopicToStorage`.